tarofreighter/chain_porter.go

```diff
@@ -463,23 +488,45 @@ func (p *ChainPorter) transferReceiverProof(pkg *sendPackage) error {
 			ctx, cancel := p.WithCtxQuitNoTimeout()
 			defer cancel()
 
-			// Before attempting to deliver the proof, log that
-			// an attempted delivery is about to occur.
-			err := p.cfg.ExportLog.LogParcelDeliveryAttempt(
+			// Query parcel delivery log to ensure a sensible
+			// rate of parcel delivery attempts.
+			timestamps, err := p.cfg.ExportLog.QueryParcelDeliveryLog(
 				ctx, pkg.OutboundPkg.NewAnchorPoint,
 			)
 			if err != nil {
-				log.Errorf("unable to log proof delivery "+
-					"attempt: %v", err)
+				err = fmt.Errorf("unable to retrieve proof delivery "+
+					"logs: %v", err)
+				errorsChan <- err
+				return
 			}
 
-			err = p.cfg.ProofCourier.DeliverProof(
-				ctx, *pkg.ReceiverAddr, receiverProof,
+			// Determine whether the past receiver proof delivery
+			// attempts occurred far enough in the past to warrant
+			// a new set of delivery attempts. Otherwise, wait.
+			timeSinceLastAttempt := timeSinceLastDeliveryAttempt(timestamps)
+			backoffResetWait := p.cfg.ReceiverProofTransferConfig.BackoffResetWait
+			err = p.wait(
```

Comment:
nit: do we want to wait if this is the first time that we are trying to send it?

Maybe something like
```
backoffResetWait := p.cfg.ReceiverProofTransferConfig.BackoffResetWait
if len(timestamps) > 0 && timeSinceLastDeliveryAttempt(timestamps) < backoffResetWait {
			err = p.wait(...)
                       // Check err
}
err = p.backoffTransferReceiverProof(ctx, pkg, receiverProof)
```

Reply:
Good catch, thanks!